job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/deployToWeblogic.groovy

```diff
@@ -0,0 +1,43 @@
+job('example') {
+    publishers {
+        deployToWeblogic {
+            // these are the default values used for optional fields, when not overridden by closure
+            mustExitOnFailure false
```

Comment:
The examples were inconsistent, some used brackets and some did not use brackets. I changed the examples so that all use brackets. For inexperienced Groovy users it's hard to guess which brackets are optional, so I just add them all to avoid confusion.